service/controller/v1/key/key.go

```diff
@@ -9,10 +9,14 @@ const (
 	NginxIngressControllerMetricPort = "10254"
 	KubeStateMetricsPort             = "10301"
 	ChartOperatorMetricPort          = "8000"
+	CertExporterMetricPort           = "9005"
+	NetExporterMetricPort            = "8000"
```

Comment:
So why are all these ports different?

Reply:
Largely historical reasons. Need to be aligned eventually.